docs/tutorials/tutorials-runtime-upgrade.md

```diff
@@ -0,0 +1,28 @@
+---
+id: tutorials-runtime-upgrade
+title: Runtime Upgrade
+sidebar_label: Runtime Upgrade
+---
+
+## How to perform runtime-upgrade
+
+* Clone <a href="https://github.com/DataHighway-DHX/node" target="_blank" class="pretty-link pretty-link-colored">node</a> repo
+
+* `cd node`
+
+* `cargo build --release`
+
+* Import the Sudo account on the https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwestlake.datahighway.com#/accounts
```

Comment:
the community who will be using documentation published at https://dev.datahighway.com/ won't have access to Sudo. they'll need guidance on how to propose changes to the runtime that'd be voted upon by the community through on-chain governance.